### ISSUE TITLE
[Feature Request] HLG support (weird type of HDR)

### ISSUE THREAD

**Issue by VP-EN** [RatingPosterDB/rpdb-folders#29]
[Feature Request] HLG support

**Is your feature request related to a problem? Please describe.**
Some TV series use the [HLG](https://en.wikipedia.org/wiki/Hybrid_log%E2%80%93gamma) format, which is sort-of HDR but sort-of it's own thing.
I'm definitely not an expert on this..

**Describe the solution you'd like**
But! It would be nice with:
1) A separate badge for it (not sure about this, as pretty much every device just registers it as HDR)
2) An option for the probing to recognize HLG and either tag it as HLG or HDR in the rpdb.json (again, not sure about what's best.. HLG would probably be most accurate)

**Additional context**
A few logs from a few episodes with the HLG:
```
S01E01: https://0bin.net/paste/zuY3IPzx#FRwsAksOMDucvf8VIsLkzypQBNq56i8auKX7MsGUhBF
S01E05: https://0bin.net/paste/xOiG6VWV#DB0u1wk2m9zsXPI6b-SoP8U3Wz06jH2X0yRBZ+hbXaF
S02E01: https://0bin.net/paste/VylBZrVH#-+JAsw0GT1IyVHIXzOfSuL4aVEbCijuWdpFDygvuuBN
S02E07: https://0bin.net/paste/agsCA4tC#-7IAF5z29BIq068xJ6gO7knkz2LMyvBHOMABHSy9AeW
```

**Comment by jaruba** [comment]
As far as I understand it, HLG is a standard, HDR is also a standard. But HLG is a subset of HDR, comparing it to DoVi which is a standard as well, but not a subset of HDR.

So a video can be considered to be both HLG and HDR, while a video cannot be considered both DoVi and HDR.

HLG is a standard that makes videos backwards compatible to HDR, so HLG is not a color range standard at all (like HDR and DoVi are), it's a compatibility standard.

In that sense I feel like an HLG badge makes less sense than using the HDR badge for such videos too..

**Comment by VP-EN** [comment]
Yeah HDR badge woud suffice IMO too

**Comment by jaruba** [comment]
Added, it will be available in `v0.1.8`

**Comment by jaruba** [comment]
@VP-EN The v0.1.8 version was just released that includes your requested feature.
